ape_aws/kms/_cli.py

```diff
@@ -54,16 +60,90 @@ def create_key(
     cli_ctx.logger.success(f"Key created successfully with ID: {key_id}")
 
 
-# TODO: Add `ape aws kms import`
+@kms.command(name="import")
+@ape_cli_context()
+@click.option(
+    "-p",
+    "--private-key",
+    "private_key",
+    multiple=False,
+    help="The private key to import",
+)
+@click.option(
+    "-a",
+    "--admin",
+    "administrators",
+    multiple=True,
+    help="Apply key policy to a list of administrators if applicable, ex. -a ARN1, -a ARN2",
+    metavar="list[ARN]",
+)
+@click.option(
+    "-u",
+    "--user",
+    "users",
+    multiple=True,
+    help="Apply key policy to a list of users if applicable, ex. -u ARN1, -u ARN2",
+    metavar="list[ARN]",
+)
+@click.option(
+    "-d",
+    "--description",
+    "description",
+    help="The description of the key you intend to create.",
+    metavar="str",
+)
+@click.argument("alias_name")
+def import_key(
+    cli_ctx,
+    alias_name: str,
+    private_key: bytes,
+    administrators: list[str],
+    users: list[str],
+    description: str,
+):
+    def ask_for_passphrase():
+        return click.prompt(
+            "Create Passphrase to encrypt account",
+            hide_input=True,
+            confirmation_prompt=True,
+        )
+
+    passphrase = ask_for_passphrase()
+    key_spec = ImportKeyRequest(
+        alias=alias_name,
+        description=description,  # type: ignore
+        admins=administrators,
+        users=users,
+    )
+    key_id = kms_client.create_key(key_spec)
+    create_key_response = kms_client.get_parameters(key_id)
+    public_key = create_key_response["PublicKey"]
+    import_token = create_key_response["ImportToken"]
+    import_key_spec = ImportKey(
+        **key_spec.model_dump(),
+        key_id=key_id,  # type: ignore
+        public_key=public_key,  # type: ignore
+        private_key=private_key,  # type: ignore
+        import_token=import_token,  # type: ignore
+    )
+    response = kms_client.import_key(import_key_spec)
+    if response["ResponseMetadata"]["HTTPStatusCode"] != 200:
+        cli_ctx.abort("Key failed to import into KMS")
+    cli_ctx.logger.success(f"Key imported successfully with ID: {key_id}")
+    aws_account_container = AwsAccountContainer(name="aws", account_type=KmsAccount)
+    aws_account_container.add_private_key(alias_name, passphrase, import_key_spec.private_key_hex)
```

Comment:
I don't think we need this method if we successfully import the key (we should discard the private key after importing)